### PR TITLE
Reacting to ApplicationServices removal from HttpContext

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Reacting to https://github.com/aspnet/HttpAbstractions/pull/487

### CODE REVIEW

**Comment by pranavkm** [comment]
:100: points for having reaction PRs ready before the change goes in.

**Comment by dougbu** [comment]
:+1: unheard of and awesome :dart:

File: test/Microsoft.AspNet.TestHost.Tests/TestServerTests.cs [hunk 1]
```diff
@@ -125,38 +125,17 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
...
-
-        [Fact]
         public async Task CanSetCustomServiceProvider()
         {
             var server = TestServer.Create(app =>
             {
                 app.Run(context =>
                 {
-                    context.ApplicationServices = new ServiceCollection()
+                   context.RequestServices = new ServiceCollection()
```
Comment (davidfowl) [inline test/Microsoft.AspNet.TestHost.Tests/TestServerTests.cs]: ohhh space

**Comment by davidfowl** [comment]
:shipit:

**Comment by JunTaoLuo** [comment]
spacing fixed on dev

**Pull request merged**